railib/api.py

```diff
@@ -879,6 +905,6 @@ def exec_async(
 get_compute = get_engine  # deprecated, use get_engine
 list_computes = list_engines  # deprecated, use list_engines
 list_edb = list_edbs  # deprecated, use list_edbs
-delete_source = delete_model  # deprecated, use delete_model
+delete_source = delete_models  # deprecated, use delete_model
```

Comment:
this changes the signature (i believe) .. so using this trick to support old names doesnt work, as its still breaking.

Reply:
oh totally missed that, yep unfortunately this a breaking change, we have two options here:
* keep supporting `delete_source` by adding an equivalent `delete_model`
* deprecate `delete_source` and remove it from the api as it is also removed from other SDKs
I think the first option makes more sense, @bradlo what do you think ?

Reply:
finally added an equivalent `delete_model`